src/components/Layout/index.tsx

```diff
@@ -1,5 +1,5 @@
-import Header from "./Header";
-import Footer from "./Footer";
+import Header from "./Header.tsx";
+import Footer from "./Footer.tsx";
```

Comment:
nit: I believe these don't need the `.tsx` ending, and I _think_ it's convention to leave it off